File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCSAPPROVEGrdPatientListInfo.java
package nta.med.data.model.ihis.ocsa;

import java.math.BigInteger;
import java.util.Date;

public class OCSAPPROVEGrdPatientListInfo {
	private Date naewonDate;
	private String gwa;
	private String gwaName;
	private String doctorName;
	private BigInteger nalsu;
	private String bunho;
	private String doctor;
	private String gubunName;
	private String chojaeName;
	private String naewonType;
	private Double jubsuNo;
	private Double pkOrder;
	private String inputGubun;
	private String toiwonDrg;
	private String inputTab;
	private String ioGubun;
	private String patientName;
	private String selectYn;

	public OCSAPPROVEGrdPatientListInfo(Date naewonDate, String gwa,
			String gwaName, String doctorName, BigInteger nalsu, String bunho,
			String doctor, String gubunName, String chojaeName,
			String naewonType, Double jubsuNo, Double pkOrder,
			String inputGubun, String toiwonDrg, String inputTab,
			String ioGubun, String patientName, String selectYn) {
		super();
		this.naewonDate = naewonDate;
		this.gwa = gwa;
		this.gwaName = gwaName;
		this.doctorName = doctorName;
		this.nalsu = nalsu;
		this.bunho = bunho;
		this.doctor = doctor;
		this.gubunName = gubunName;
		this.chojaeName = chojaeName;
		this.naewonType = naewonType;
		this.jubsuNo = jubsuNo;
		this.pkOrder = pkOrder;
		this.inputGubun = inputGubun;
		this.toiwonDrg = toiwonDrg;
		this.inputTab = inputTab;
		this.ioGubun = ioGubun;
		this.patientName = patientName;
		this.selectYn = selectYn;
	}

	public Date getNaewonDate() {
		return naewonDate;
	}

	public void setNaewonDate(Date naewonDate) {
		this.naewonDate = naewonDate;
	}

	public String getGwa() {
		return gwa;
	}

	public void setGwa(String gwa) {
		this.gwa = gwa;
	}

	public String getGwaName() {
		return gwaName;
	}

	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}

	public String getDoctorName() {
		return doctorName;
	}

	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}

	public BigInteger getNalsu() {
		return nalsu;
	}

	public void setNalsu(BigInteger nalsu) {
		this.nalsu = nalsu;
	}

	public String getBunho() {
		return bunho;
	}

	public void setBunho(String bunho) {
		this.bunho = bunho;
	}

	public String getDoctor() {
		return doctor;
	}

	public void setDoctor(String doctor) {
		this.doctor = doctor;
	}

	public String getGubunName() {
		return gubunName;
	}

	public void setGubunName(String gubunName) {
		this.gubunName = gubunName;
	}

	public String getChojaeName() {
		return chojaeName;
	}

	public void setChojaeName(String chojaeName) {
		this.chojaeName = chojaeName;
	}

	public String getNaewonType() {
		return naewonType;
	}

	public void setNaewonType(String naewonType) {
		this.naewonType = naewonType;
	}

	public Double getJubsuNo() {
		return jubsuNo;
	}

	public void setJubsuNo(Double jubsuNo) {
		this.jubsuNo = jubsuNo;
	}

	public Double getPkOrder() {
		return pkOrder;
	}

	public void setPkOrder(Double pkOrder) {
		this.pkOrder = pkOrder;
	}

	public String getInputGubun() {
		return inputGubun;
	}

	public void setInputGubun(String inputGubun) {
		this.inputGubun = inputGubun;
	}

	public String getToiwonDrg() {
		return toiwonDrg;
	}

	public void setToiwonDrg(String toiwonDrg) {
		this.toiwonDrg = toiwonDrg;
	}

	public String getInputTab() {
		return inputTab;
	}

	public void setInputTab(String inputTab) {
		this.inputTab = inputTab;
	}

	public String getIoGubun() {
		return ioGubun;
	}

	public void setIoGubun(String ioGubun) {
		this.ioGubun = ioGubun;
	}

	public String getPatientName() {
		return patientName;
	}

	public void setPatientName(String patientName) {
		this.patientName = patientName;
	}

	public String getSelectYn() {
		return selectYn;
	}

	public void setSelectYn(String selectYn) {
		this.selectYn = selectYn;
	}

}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCSAPPROVEGrdPatientListInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OCSAPPROVEGrdPatientListInfo
	{
		private String _naewonDate;
		private String _gwa;
		private String _gwaName;
		private String _doctorName;
		private String _nalsu;
		private String _bunho;
		private String _doctor;
		private String _gubunName;
		private String _chojaeName;
		private String _naewonType;
		private String _jubsuNo;
		private String _pkOrder;
		private String _inputGubun;
		private String _toiwonDrg;
		private String _inputTab;
		private String _ioGubun;
		private String _patientName;
		private String _selectYn;

		public String NaewonDate
		{
			get { return this._naewonDate; }
			set { this._naewonDate = value; }
		}

		public String Gwa
		{
			get { return this._gwa; }
			set { this._gwa = value; }
		}

		public String GwaName
		{
			get { return this._gwaName; }
			set { this._gwaName = value; }
		}

		public String DoctorName
		{
			get { return this._doctorName; }
			set { this._doctorName = value; }
		}

		public String Nalsu
		{
			get { return this._nalsu; }
			set { this._nalsu = value; }
		}

		public String Bunho
		{
			get { return this._bunho; }
			set { this._bunho = value; }
		}

		public String Doctor
		{
			get { return this._doctor; }
			set { this._doctor = value; }
		}

		public String GubunName
		{
			get { return this._gubunName; }
			set { this._gubunName = value; }
		}

		public String ChojaeName
		{
			get { return this._chojaeName; }
			set { this._chojaeName = value; }
		}

		public String NaewonType
		{
			get { return this._naewonType; }
			set { this._naewonType = value; }
		}

		public String JubsuNo
		{
			get { return this._jubsuNo; }
			set { this._jubsuNo = value; }
		}

		public String PkOrder
		{
			get { return this._pkOrder; }
			set { this._pkOrder = value; }
		}

		public String InputGubun
		{
			get { return this._inputGubun; }
			set { this._inputGubun = value; }
		}

		public String ToiwonDrg
		{
			get { return this._toiwonDrg; }
			set { this._toiwonDrg = value; }
		}

		public String InputTab
		{
			get { return this._inputTab; }
			set { this._inputTab = value; }
		}

		public String IoGubun
		{
			get { return this._ioGubun; }
			set { this._ioGubun = value; }
		}

		public String PatientName
		{
			get { return this._patientName; }
			set { this._patientName = value; }
		}

		public String SelectYn
		{
			get { return this._selectYn; }
			set { this._selectYn = value; }
		}

		public OCSAPPROVEGrdPatientListInfo() { }

		public OCSAPPROVEGrdPatientListInfo(String naewonDate, String gwa, String gwaName, String doctorName, String nalsu, String bunho, String doctor, String gubunName, String chojaeName, String naewonType, String jubsuNo, String pkOrder, String inputGubun, String toiwonDrg, String inputTab, String ioGubun, String patientName, String selectYn)
		{
			this._naewonDate = naewonDate;
			this._gwa = gwa;
			this._gwaName = gwaName;
			this._doctorName = doctorName;
			this._nalsu = nalsu;
			this._bunho = bunho;
			this._doctor = doctor;
			this._gubunName = gubunName;
			this._chojaeName = chojaeName;
			this._naewonType = naewonType;
			this._jubsuNo = jubsuNo;
			this._pkOrder = pkOrder;
			this._inputGubun = inputGubun;
			this._toiwonDrg = toiwonDrg;
			this._inputTab = inputTab;
			this._ioGubun = ioGubun;
			this._patientName = patientName;
			this._selectYn = selectYn;
		}

	}
}